dev/provider_packages/SETUP_TEMPLATE.cfg.jinja2

```diff
@@ -67,9 +67,6 @@ python_requires = ~=3.7
 packages = find:
 setup_requires = {{ SETUP_REQUIREMENTS }}
 install_requires = {{ INSTALL_REQUIREMENTS }}
-    gitpython
-    wheel
-
```

Comment:
was already removed in https://github.com/apache/airflow/pull/22382/files

Reply:
Yep. need to rebase. I will also likely add another fix in cncf.kubernetes while we are at this. BTW.

Reply:
The fix is in #22385 so once this is merged, I will rebase and prpare the new release.